### ISSUE TITLE
fire_hydrant number for Germany

### ISSUE THREAD

**Issue by Elsensee** [streetcomplete/StreetComplete#5272]
I've seen that the `AddFireHydrantRef` quest is only enabled for Switzerland and France. I would request for it to be enabled for Germany as well.

**Use case**
I started tagging missing fire hydrants and found out that some have "ref" numbers tagged and some don't. After looking at the sign, I saw that there is a corresponding number to this. It's number 6 on the sign: https://wiki.openstreetmap.org/wiki/File:Hydrantenschild_mit_Legende.svg

I've only seen very few without any such number given on the sign.

**Proposed Solution**
Enable the quest for Germany as well. Probably even add a guiding image for entering it like with the diameter.

**Comment by qugebert** [comment]
Some reference (the issue, where this quest was discussed)
https://github.com/streetcomplete/StreetComplete/issues/3059
And in the pull request @mcliquid explained why it is not enabled in germany
https://github.com/streetcomplete/StreetComplete/pull/4440

And at least in Bavaria, i've never seen any signed reference number for fire hydrants.

**Comment by Elsensee** [comment]
I did find the first issue but it was closed and I don't like to discuss closed issues that much.

The second, I did not find. It's interesting that it seems to vary by state? In Hamburg they appear on every sign.
Wikipedia also mentions that they might not exist in every area: https://de.wikipedia.org/wiki/Hinweisschilder_zu_Stra%C3%9Feneinbauten#Hinweisschilder_f%C3%BCr_Hydranten (The images below also show that some signs have a number, some don't)

Googling "Hydrantennummer" yields some results, which confirms that this definitely is a regional thing.

Doing a quick overpass query shows that even in southern germany some fire_hydrant's have a numeric ref, while others have a text like "Im Garten", etc.

Sooo... I don't know?
